bin/evm2near/src/compile.rs

```diff
@@ -73,10 +73,6 @@ pub fn compile<'a>(
     compiler.emit_abi_execute();
     let abi_data = compiler.emit_abi_methods(input_abi).unwrap();
 
-    let table = compiler.builder.tables.get_mut(0).unwrap();
```

Comment:
havent found any reasons to keep that. is it obsolete?

Reply:
If the tests pass then it's probably not needed anymore.